13-Tenant Security Solution Setup/Scripts/RemediationScripts/Remediate-ConfigASCTier.ps1

```diff
@@ -0,0 +1,389 @@
+function Pre_requisites
+{
+    <#
+    .SYNOPSIS
+    This command would check pre requisities modules.
+    .DESCRIPTION
+    This command would check pre requisities modules to perform remediation.
+	#>
+
+    Write-Host "Required modules are: Az.Resources, Az.Security, Az.Account" -ForegroundColor Cyan
+    Write-Host "Checking for required modules..."
+    $availableModules = $(Get-Module -ListAvailable Az.Resources, AzureAD, Az.Accounts)
+    
+    # Checking if 'Az.Accounts' module is available or not.
+    if($availableModules.Name -notcontains 'Az.Accounts')
+    {
+        Write-Host "Installing module Az.Accounts..." -ForegroundColor Yellow
+        Install-Module -Name Az.Accounts -Scope CurrentUser
+    }
+    else
+    {
+        Write-Host "Az.Resources module is available." -ForegroundColor Green
```

Comment:
In the if block we are checking for Az.Accounts and in else block we are showing Az.Resource is available.

Reply:
Updated